crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -96,11 +96,17 @@ fn vertex(vertex_no_morph: Vertex) -> VertexOutput {
 #endif // VERTEX_UVS_B
 
 #ifdef NORMAL_PREPASS_OR_DEFERRED_PREPASS
+#ifdef VERTEX_NORMALS
+    var vertex_normal = vertex.normal;
+#else // VERTEX_NORMALS
+    var vertex_normal = vec3(0.0);
+#endif // VERTEX_NORMALS
+
 #ifdef SKINNED
-    out.world_normal = skinning::skin_normals(world_from_local, vertex.normal);
+    out.world_normal = skinning::skin_normals(world_from_local, vertex_normal);
 #else // SKINNED
     out.world_normal = mesh_functions::mesh_normal_local_to_world(
-        vertex.normal,
+        vertex_normal,
```

Comment:
I don't really think it makes sense to pass it a vec3 with zeroes. If the mesh has no normals it shouldn't try to do anything with them

Reply:
Yeah that's fair, I've surrounded the whole block with `#ifdef VERTEX_NORMALS` instead of calling it with a default value.